server/elm/src/Pages/Scoreboard/Utils.elm

```diff
@@ -0,0 +1,5 @@
+module Pages.Scoreboard.Utils exposing (..)
+
+
+tmp =
+    0
```

Comment:
What's the purpose of this? Do we use it somewhere?

Reply:
@AronNovak this PR lays infrastructure for the feature.
Right now, there're no utility funcitons, but very likely that we'll  need this file when developing the feature.
Elm does not allow empty file, so I put there something temporary for now.